qa/automation/CareerPortal.Tests.Scenarios/3. Send_CV/Send_CV.feature

```diff
@@ -0,0 +1,148 @@
+Feature: Send_CV
+
+Background:
+    Given user visited the site https://career.quantori.com/ru
+
+Scenario Outline: Correct CV submission
+    Given user entered a <name>, <email>, <message>, attached SV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then CV should be successfully submitted
+
+    Examples:
+        | name        | email                    | message     |
+        | Egor  Berdin| egor.berdin@quantori.com | Hello_world |
+
+
+
+Scenario Outline: Correct CV submission without message
+    Given user entered a <name>, <email>, attached SV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then CV should be successfully submitted
+
+    Examples:
+        | name       | email                    |
+        | Egor Berdin| egor.berdin@quantori.com |
+ 
+
+
+Scenario Outline: CV submission without Name
+    Given user entered an <email>, attached CV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then under the field "Name" a warning appears "This field is required"
+
+    Examples:
+        | email                    |
+        | egor.berdin@quantori.com |
+
+
+
+Scenario Outline: CV submission without Email
+    Given user entered a <name>, attached CV (PDF less than 10 MB), checked the checkbox
+    When user clicks on a button "Send"
+    Then under the field "Email" a warning appears "This field is required"
+
+    Examples:
+        | name         |
+        | Egor  Berdin |
+
+
+
+Scenario Outline: CV submission with 501 characters long message
+    Given user entered a <name>, <email>, attached CV (PDF less than 10 MB), checked the checkbox
+    And writed a message 501 characters long
+    When user clicks on a button "Send"
+    Then under the field "Message" a warning appears "The maximum number of characters is 500"
+
+    Examples:
+        | name         | email                    |
+        | Egor  Berdin | egor.berdin@quantori.com |
+
+
+
+Scenario Outline: CV submission with 500 characters long message
+    Given user entered a <name>, <email>, attached CV (PDF less than 10 MB), checked the checkbox
+    And writed a message 500 characters long
+    When user clicks on a button "Send"
+    Then CV should be successfully submitted
+
+    Examples:
+        | name         | email                    |
+        | Egor  Berdin | egor.berdin@quantori.com |
+
+
+
+Scenario Outline: CV submission without checked the checkbox
```

Comment:
If you would like to test every field, you should unite such scenario into Scenario Outline. 
Also, you should apply test design technique to find the best combination.

It can be looks like this; 
Scenario Outline: User can't sent CV without filled one of mandatory field 
When user inputs <Name>
And user inputs <Email>
And user uploads <Attachment>
And user makes checkbox in <Checkbox State>
Then CV should not be sent 
And <Error message> message should be visible 
Examples: 
|Name          | Email |Checkbox state| Attachment| Error message|
|                    |test@mail.com|selected| Attatchment1|This field is required|
|Test Name   |                        |selected| Attachment1|This field is required|

etc.